### ISSUE TITLE
Isomorphism vs bijection

### ISSUE THREAD

**Issue by aljazerzen** [abuseofnotation/category-theory-illustrated#33]
If I an not wrong an isomorphism is "structure-preserving bijection" i.e. "bijective homomorphism".

First chapter describes isomorphisms in context of plain sets that don't have structure. This is technically correct, but it may be more appropriate to use term "bijection" instead.

(I'm sorry if this is explained later in the book)

**Comment by abuseofnotation** [comment]
Yes, it is my impression as well, that two terms are roughly equivalent.

Why do you think "bijection" is more appropriate than "isomorphism"?

**Comment by aljazerzen** [comment]
Isomorphism is (to my knowledge) used in context of monoids and groups which have structure. It implies two things:
- the function is bijective (injective and surjective i.e. invertible),
- the function is homomorphism (that is: `f(a) * f(b) = f(a * b)`, where `*` is the group's operator).

So all isomorphism are bijective, but not all bijections are isomorphic.

In the case of first chapter, sets don't even have the operation, which means it does not really make sense to talk about them being homomorphic, and by extension, isomorphic.

**Comment by abuseofnotation** [comment]
An isomorphism is a "structure-preserving" function. It can mean different things for different structures. 

In the case of sets,  bijection is the same thing as isomorphism simply because sets have no structure. (https://math.stackexchange.com/questions/90923/isomorphism-of-sets)

So, in this context, both words are equivalent, but since my aim is not to talk about sets specifically, but to generalize the notion of isomorphism for other objects, I use "isomorphism". If you want to know where I am going with this, skip to the last chapter:
https://abuseofnotation.github.io/category-theory-illustrated/06_functors/#categorical-isomorphisms
